website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -227,7 +227,7 @@ resource "azurerm_kubernetes_cluster" "example" {
 
 !> **Note:** A migration scenario from `service_principal` to `identity` is supported. When upgrading `service_principal` to `identity`, your cluster's control plane and addon pods will switch to use managed identity, but the kubelets will keep using your configured `service_principal` until you upgrade your Node Pool.
 
-* `sku_tier` - (Optional) The SKU Tier that should be used for this Kubernetes Cluster. Possible values are `Free`, `Paid` and `Standard` (which includes the Uptime SLA). Defaults to `Free`.
+* `sku_tier` - (Optional) The SKU Tier that should be used for this Kubernetes Cluster. Possible values are `Free`, and `Standard` (which includes the Uptime SLA). Defaults to `Free`.
```

Comment:
We should call out that `Paid` was a previous value and how to migrate from it, in addition to in the changelog? Something like this:

```suggestion
* `sku_tier` - (Optional) The SKU Tier that should be used for this Kubernetes Cluster. Possible values are `Free`, and `Standard` (which includes the Uptime SLA). Defaults to `Free`.

-> **NOTE:** Whilst the AKS API previously supported the `Paid` SKU - the AKS API introduced a breaking change in API Version `XXX` where the value `Paid` must now be set to `Standard`.
```